walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,58 @@
 
 package com.google.sps;
 
+import java.util.Comparator;
 import java.util.Collection;
+import static java.util.Collections.disjoint;
+import static java.util.Collections.sort;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
You want to javadoc what this method does...which is a lot.  The reader shouldn't have to read through the code to know what it does.  

you also probably want to introduce some helper methods to help break down what exactly you're doing.  The for loop might be a good candidate.

Reply:
Done.